packages/engine-server/src/markdown/remark/dendronPub.ts

```diff
@@ -208,6 +209,12 @@ function plugin(this: Unified.Processor, opts?: PluginOpts): Transformer {
           }
         }
       }
+      if (node.type === DendronASTTypes.BLOCK_ANCHOR) {
+        const procOpts = proc.data("procOpts") as any;
```

Comment:
we should use `getProcOpts` [method](https://github.com/dendronhq/dendron/blob/integ-publish/packages/engine-server/src/markdown/utils.ts#L198:L198) for this